### PR TITLE
Adds more UI modifications related to chat, and connects chat to the topic list

### PR DESCRIPTION
**Pull request by ottpeter**

### CODE REVIEW

File: src/components/ChatInput/ChatInput.tsx [hunk 1]
```diff
@@ -37,14 +41,30 @@ const ChatInput: React.FC<ChatInputProps> = ({
...
+      }
+
       await chat.registerUser(topic, { participant: ownAddress, key: privKey, stamp, nickName: nickname })
         .then(() => console.info(`user reconnected.`))
         .catch((err) => console.error(`error when reconnecting ${err.error}`));
+
+        do {
+          await waitOneRound(1000);
+          console.log("isRegistered: ", chat.isRegistered)
+        } while (!chat.isRegistered(ownAddress) && rounds < 60);
```
Comment (bosi95) [inline src/components/ChatInput/ChatInput.tsx:56]: 60: magic number. is this an empirical value ?

File: src/pages/Chat/Chat.tsx [hunk 2]
```diff
@@ -76,35 +134,62 @@ const Chat: React.FC<ChatProps> = ({
...
+  }, [currentThread]);
+
 
   return (
     <div className="chat-page">
       <Back 
-        where={"Home"}
-        link={"/home-"}
+        where={currentThread ? "Chat" : originatorPage}
+        link={currentThread ? '/home' : originatorPageUrl}
```
Comment (bosi95) [inline src/pages/Chat/Chat.tsx:150]: use ROUTES.HOME

File: src/pages/Chat/Chat.tsx [hunk 3]
```diff
@@ -76,35 +134,62 @@ const Chat: React.FC<ChatProps> = ({
...
+    setVisibleMessages([...newlyFilteredMessages]);
+  }, [currentThread]);
+
 
   return (
     <div className="chat-page">
       <Back 
-        where={"Home"}
-        link={"/home-"}
+        where={currentThread ? "Chat" : originatorPage}
```
Comment (bosi95) [inline src/pages/Chat/Chat.tsx:149]: could be better naming for back button

**Review by bosi95 — APPROVED** [review]

File: src/components/ChatInput/ChatInput.tsx (hunk 1 above)
Comment (ottpeter) [reply src/components/ChatInput/ChatInput.tsx:56]: 60 should be 1 minute

File: src/pages/Chat/Chat.tsx (hunk 3 above)
Comment (ottpeter) [reply src/pages/Chat/Chat.tsx]: Renamed to "Back to main thread"

**Pull request merged**